app/interactors/rake_interactor.rb

```diff
@@ -31,6 +33,8 @@ def call
       File.open(@log_file, 'a') do |f|
         f.puts log_buffer.string
       end
+
+      Rake::Task.clear
```

Comment:
Ensure that we clear the loaded rake tasks afterwards so we don't reload it more than once.